docs/notary/setup-Komodo-Notary-Node.md

```diff
@@ -34,19 +35,18 @@ We recommend the Notary Node Operators to check the Table at [https://github.com
 _To notarise KMD -> LTC, the `-notary=".litecoin/litecoin.conf"` flag is required when launching KMD._
 
 ::: tip Note
-You should have at least 64GB RAM and 32 GB [swap space](https://www.digitalocean.com/community/tutorial_collections/how-to-add-swap-space) on your main server to avoid daemons crashing due to OOM.
+You should have at least 64GB RAM and 32 GB [swap space](https://www.digitalocean.com/community/tutorial_collections/how-to-add-swap-space) on your main server to avoid daemons crashing due to OOM errors.
```

Comment:
sure that 64GB is enough? i took the values from my NN that shows the lowest ones:
```
MiB Mem : 128914.4 total,   2105.7 free,  83527.1 used,  43281.6 buff/cache
MiB Swap: 100348.0 total,  90502.6 free,   9845.4 used.  38393.4 avail Mem
```
the others show up to 97GB used

Reply:
Mine seem close, but ok.
```
smk762@dragonhound:~$ free -h
              total        used        free      shared  buff/cache   available
Mem:           62Gi        60Gi       503Mi       0.0Ki       1.9Gi       1.7Gi
Swap:          48Gi        28Gi        19Gi
```
```
smk762@devHound:~$ free -h
              total        used        free      shared  buff/cache   available
Mem:           62Gi        60Gi       453Mi       0.0Ki       1.6Gi       1.3Gi
Swap:          32Gi        25Gi       7.0Gi
```